examples/PoCTutorial/model/PoC Tutorial.esdl

```diff
@@ -222,7 +222,7 @@
         <port xsi:type="esdl:InPort" connectedTo="ca90e0a5-f866-4ec2-9b3b-07f054f1c2b2" id="35ddd90a-e45c-4afd-95b4-80ce6c927071" carrier="9f6aeb1a-138b-4bb9-9a09-d524e94658e6_ret" name="In_ret"/>
         <port xsi:type="esdl:OutPort" id="6f9268a0-1fd9-42f6-8821-2d7d4a6e5618" connectedTo="6b4d9bba-484b-46aa-bfe6-895d491b6747" carrier="9f6aeb1a-138b-4bb9-9a09-d524e94658e6_ret" name="Out_ret"/>
       </asset>
-      <asset xsi:type="esdl:ResidualHeatSource" power="50000000.0" name="ResidualHeatSource_76f0" id="76f0817c-9f01-431e-be96-dbf3ee806d76" state="OPTIONAL">
+      <asset xsi:type="esdl:ResidualHeatSource" technicalLifetime="30.0" power="50000000.0" name="ResidualHeatSource_76f0" id="76f0817c-9f01-431e-be96-dbf3ee806d76" state="OPTIONAL">
```

Comment:
If no technical lifetime is added in the ESDL, in MESIDO the default is already set to 30 years, so to test properly, pick a different lifetime.

Reply:
Agree. Value changed from 30 to 15 years